load.go

```diff
@@ -65,6 +67,40 @@ func loadSourceStructs(path string, tags string, handleErr handlePkgLoadErr) (so
 		cfg.BuildFlags = []string{fmt.Sprintf("-tags=%s", tags)}
 	}
 
+	var glob string
+	if strings.Contains(path, "*") {
+		path, glob = stdpath.Split(path)
+		path = path[0 : len(path)-1] // remove trailing slash
```

Comment:
```suggestion
		path = strings.TrimSuffix(path, "/") // remove trailing slash
```
not sure this is safe, path could be empty.